numpy/tests/test_matlib.py

```diff
@@ -56,3 +57,6 @@ def test_repmat():
     y = np.array([[0, 1, 2, 3, 0, 1, 2, 3],
                   [0, 1, 2, 3, 0, 1, 2, 3]])
     assert_array_equal(x, y)
+
+def test_dirichlet():
+    dirichlet([5, 9, 0, 8])
```

Comment:
```suggestion
def test_dirichlet():
    y = dirichlet([5, 9, 0, 8])
    assert y[2] == 0
```
We don't just want to that the function doesn't throw an error, we also want to show that the function returns the right result when 0 is given as input.